backend/src/core/middlewares/file-attachment.middleware.ts

```diff
@@ -37,14 +42,32 @@ function preprocessPotentialIncomingFile(
      * express-fileupload does not throw error if num files
      * exceeded, instead truncates array to specified num
      */
+    // same as comment above; unlogged + status
     if (req.body.attachments.length > FILE_ATTACHMENT_MAX_NUM) {
       res.status(413).json({ message: 'Number of attachments exceeds limit' })
     }
   }
   next()
 }
 
+/*
+ * Upload attached files to S3 and return the S3 keys?
+ * */
+
+// function uploadAttachments = async (
+// req: Request,
+//   res: Response,
+//   next: NextFunction
+// ) {
+//   if (req.files?.attachments) {
+//     const attachments = req.files.attachments as fileUpload.UploadedFile[]
+//     await FileAttachmentService.uploadAttachments(attachments)
+//   }
+//   next()
+// }
+
 export const FileAttachmentMiddleware = {
   fileUploadHandler,
   preprocessPotentialIncomingFile,
+  // uploadAttachments,
```

Comment:
I dont think we need to save the actual file on to S3 but rather just keep the original file name in a column of our DB as an audit trail of the file being sent over.

Reply:
resolved → as discussed, we'll save a hash of the file attachment as a reasonable compromise between accountability and privacy.